mitodl/open-discussions
# Add channel_type to Channel model

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #1483 

#### What's this PR do?
Adds `channel_type` to `Channel`, adds logic to keep it updated and a management command to backpopulate existing `Channel` objects

#### How should this be manually tested?
Run the management command and verify that `Channel.objects.values_list('channel_type')` has values which look right
